ArmaForces.Boderator/ArmaForces.Boderator.BotService/Discord/DependecyInjection/IDiscordServiceBuilder.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ArmaForces.Boderator.BotService.Discord.DependencyInjection
+{
+    internal interface IDiscordServiceBuilder
+    {
```

Comment:
Why an empty interface?

Reply:
For future configuration parameters like:
```cs
services.AddDiscordService(Helpers.Configuration.DiscordToken).WithStatusRoller()
```